### DISCUSSION TITLE
Stackchain Terminology Clarification

### DISCUSSION THREAD

**Discussion by FREEBORN-MAN** [stackchainsats/stackchain#19] [General]
I am looking through the Glossary of Terms and had a couple thoughts/questions I'd like to discuss. Bare with me here because some of these questions seem simple but I'm trying to work out some misunderstandings I have. 

I want to set a good example for new stackchainers and also give my blocks the best opportunity to be included in the chain.

**"BLOCK: A reply that contains one or more receipts"**
-So by definition, a block must contain a receipt and no other information required? (Y/N)
-Do we accept a block with HASHTAG StackChainTip? We've previously told people not to self-validate and I would consider this self validating. We've got into situations where two competing chains are moving quick and one side is not self-validating and the other is posting the hashtag in every buy. This one chain shows up on the search quicker regardless if the block was "valid" or checked by a node. I believe this gives certain peoples "buys" more attention even if they deserve more scrutiny. 

**"RECEIPT: A screenshot of a stack"**

1. Does a link posted in the proposed "block" count as a receipt? (Y/N)
- A link is technically not a receipt but I like that we accept these blocks. Adding wording like "a link to a receipt is acceptable" would clear this up for me. 
2. What are the requirements for that link? 
- Do we require it to be a twitter post / link or can it be to another website? (must be twitter / can be anything but it will draw more scrutiny)
- Assuming the link is to another Twitter post, can that linked twitter post be located anywhere within twitter? (Y/N)
3. What are the requirements for the receipt? here are some common requirements I've seen enforced
- must contain FIAT denomination OR sat count within ~10k sats so nodes can do the conversion with a ~20$ rounding error. 
- preferable to be from commonly recognized exchange but not a show-stopper if not
- specific transaction info is not required ie addresses, exchange transaction ID's, etc
- timestamps of buys are NOT required? Timestamps have been enforced occasionally but not always, I just want to be consistent or know when a timestamp is required

**"STACK: A purchase of Bitcoin"**
- Are we defining a stack as a purchase on specific exchanges? Obviously Strike is acceptable...but is Bisq? Are other means of P2P trading acceptable?
- Are we defining a stack as a purchase of Bitcoin with FIAT only? (Y/N) ie is trading a car P2P acceptable?

NEW PROPOSED TERM - "OVERSTACK: An attempt to mine a block with USD value over current block height"
- We use this term and reject blocks based on this but what how strict are we being here? Just based off experience 11 cent overstack is commonly acceptable but a 100$ overstack is less accepted. Where is this line? Some exchanges take fee's from what you want to purchase and it makes the final USD amount un-even. Trying to hit the EXACT sat amount to get the proper height can be difficult with an exchange that updates it's price every 5 seconds. Maybe we accept transaction fees in the total just like an exchange spreads are accepted. Unclear for me.

**Comment by von-Hodl** [comment]
We have stacks in the chain that were purchased on bisque. Also there was a cash stack. The receipt was a video of the exchange.
